woocommerce/wc-admin
# Screen readers: read date instead of data keys in 'time-comparison' bar charts

Fixes #1543.

Improve the text read when navigating `time-comparison` bar charts with a screen reader.

### Accessibility


### Screenshots
![image](https://user-images.githubusercontent.com/3616980/52863664-bcfae680-3138-11e9-8714-5c9903736593.png)

### Detailed test instructions:
- Go to the _Revenue_ report.
- Change to a bar chart.
- Use a screen reader + keyboard to navigate the bars.
- When focusing a bar, verify it reads the date + value (something like `February 3, 2018 1,648.00€`).
- Switch the chart interval to `hour` (you might need to change the selected date range).
- Verify it reads the period + hour + value (something like: `Previous Year (Feb 15, 2018) 10PM 723.00€`).
